features/networking/multus.feature

```diff
@@ -600,7 +598,7 @@ Feature: Multus-CNI related scenarios
       | namespace isolation |
       | violat              |
     """
- 
+    
```

Comment:
maybe a `sed -i -e 's/[[:space:]]\+$//g'`  to clean up trailing whitespace.  

or if vim,  `:%s/\s\+$//g`  and maybe `set list` to show the trailing whitespace.
```
set list
set listchars=tab:»\ ,trail:¬
```